mne/channels/channels.py

```diff
@@ -1007,11 +1011,52 @@ def _ch_neighbor_connectivity(ch_names, neighbors):
     ch_connectivity = np.eye(len(ch_names), dtype=bool)
     for ii, neigbs in enumerate(neighbors):
         ch_connectivity[ii, [ch_names.index(i) for i in neigbs]] = True
-
     ch_connectivity = sparse.csr_matrix(ch_connectivity)
     return ch_connectivity
 
 
+def compute_connectivity(info, ch_type):
```

Comment:
I think `compute_connectivity` is a bit misleading - I would prefer `compute_adjacency` or `find_neighbors` (if mne endorses US spelling). WDYT?